packages/account/src/Components/api-token/_tests_/api-token.spec.tsx

```diff
@@ -1,19 +1,15 @@
 import React from 'react';
 import { act, fireEvent, render, screen, waitFor } from '@testing-library/react';
-import { getPropertyValue, isDesktop, isMobile, useIsMounted, WS } from '@deriv/shared';
+import { getPropertyValue, useIsMounted, WS, isMobile } from '@deriv/shared';
 import ApiToken from '../api-token';
 import { StoreProvider, mockStore } from '@deriv/stores';
+import { FormikValues } from 'formik';
 
 jest.mock('@deriv/shared', () => ({
     ...jest.requireActual('@deriv/shared'),
     getPropertyValue: jest.fn(() => []),
-    isDesktop: jest.fn(() => true),
     isMobile: jest.fn(() => false),
```

Comment:
In the compoennt, We are using is_mobile from ui- store. Why do we need this mock?

Reply:
Some components like `ApiTokenTable` still use `isMobile() ` instead of using stores. Thus, we need this mock

Reply:
since this is unit test for ApiToken, shouldnt we mock ApiTokenTable ?

Reply:
I think unit test case for ApiTokenTable are handled in a diff testcase file